pkg/cain/cqlsh.go

```diff
@@ -124,10 +124,16 @@ func GetMaterializedViews(iK8sClient interface{}, namespace, container, pod, key
 }
 
 // Cqlsh executes cqlsh -e 'command' in a given pod
-func Cqlsh(iK8sClient interface{}, namespace, pod, container string, command []string) ([]byte, error) {
+func Cqlsh(iK8sClient interface{}, namespace, pod, container string, command []string, username string, password string) ([]byte, error) {
 	k8sClient := iK8sClient.(*skbn.K8sClient)
 
-	command = append([]string{"cqlsh", "-e"}, command...)
+	baseCommand := []string{"cqlsh"}
+	if len(username) > 0 && len(password) > 0 {
```

Comment:
the default values are set to `cassandra`. is there a case when a user passes an empty string on purpose?

Reply:
The way I wrote this part it would mean that no username/password flags would be appended.

What would you think is the best approach for this scenario? Should I refactor it to always append the username and password? Or remove the defaults, and append the flags only if a username and/or password is set?

Reply:
i think this approach makes sense!